Wallet connect and disconnect template (#29)
scale-it/algo-builder-templates: wallet-connect/README.md

# Wallet-Connect Template

> This tutorial will guide you on AlgoSigner wallet connect and disconnect.

Reviewer: ```suggestion
This template shows how to connect a AlgoSigner wallet with a Vue.js app.
```

Author: Done

> # Steps

Reviewer: ```suggestion
# Functionality
```

### `Connect`

Click on Connect button to connect to AlgoSigner. This will dislay wallet address below on successful connection.

### `Disconnect`

Click on Disconnect button to disconnect to AlgoSigner. This will remove wallet address below Connect button and disconnect from wallet.

> ### `yarn dev`

Reviewer: ```suggestion
## Building

### `yarn dev`
```

Runs the react app in the development mode.\
Open [http://localhost:8080](http://localhost:8080) to view it in the browser.

> 

Reviewer 2: Instead of react shouldn't it say vue ..?

Author: Done

The page will reload if you make edits.\
You will also see any lint errors in the console.

### `yarn test`

Launches the test runner in the interactive watch mode.\
See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.

### `yarn build`

Builds the app for production to the `build` folder.\
It correctly bundles React in production mode and optimizes the build for the best performance.

> 

Reviewer 2: ditto

Author: Done

> # Wallet-Connect Template

Reviewer: ```suggestion
# Wallet Template
```

This template shows how to connect a AlgoSigner wallet with a Vue.js app.

> # Config

Reviewer: ```suggestion
## Config
```

Reviewer: I was already writing about this: only the title should have level one header.

The file present at `/src/constant/index.ts` is used for ledger name (MainNet, TestNet, Private-Net). By default it is being set to MainNet.

> # Supported Wallets

Reviewer: ```suggestion
## Supported Wallets
```

The follwing wallets are supported:

- AlgoSigner
- My Algo Connect
- Wallet Connect

> # Functionality

Reviewer: ```suggestion
## Functionality
```
